tutorials/builtin_algorithms/GCP_AI_Platform_Image_Classification_Built_in_Algorithm.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<p>Does this step require the user to already have a TPU service account? What if the user is using a new project - are there any additional set up steps such as requesting quota?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ml-on-gcp/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>